tests/TestCurrencyHelper.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace PodPoint\I18n\Tests;
+
+use Illuminate\Config\Repository;
+use PHPUnit\Framework\TestCase;
+use PodPoint\I18n\CurrencyCode;
+use PodPoint\I18n\CurrencyHelper;
+
+class TestCurrencyHelper extends TestCase
+{
+    /**
+     * Data provider for testToFormat.
+     *
+     * @return array
+     */
+    public function providerTestToFormat()
+    {
+        $value = 1500.5;
+
+        return [
+            'pound sterling in english' => [
+                $value,
+                'en',
+                CurrencyCode::POUND_STERLING,
+                '£1,500.50',
+            ],
+            'pound sterling in norwegian' => [
+                $value,
+                'no',
+                CurrencyCode::POUND_STERLING,
+                '£ 1 500,50',
```

Comment:
is this really how they would display pounds? 🤔

Reply:
Yep I tested it... I agree it's weird because personally I would expect it to be `1 500,50 £` but 🤷‍♂️

Reply:
oh is that how they display it in France? that's even weirder!

Reply:
Nope :-) just using this https://documentation.progress.com/output/rb/doc/index.html#page/rb/currency-formats.html